src/test/java/api/support/APITests.java

```diff
@@ -188,7 +191,6 @@ public void beforeEach()
       useDefaultRollingPolicyCirculationRules();
     }
 
-    configureDefaultTimezone();
```

Comment:
Can you help me understand why this was removed?

I'm aware that the default is UTC, and so this is technically redundant, I'd prefer it was explicitly set.

(I'd also prefer that the system fail if this wasn't configured, but I've accepted that is out of scope for this work at the moment)

Reply:
yes, right, by default we have UTC default time zone. 
I used these settings in tests for the default time zone, custom, and UTC values